### ISSUE TITLE
Sometimes only peerId is passed to libp2p dialer

### ISSUE THREAD

**Issue by EmiM** [TryQuiet/quiet#2027]
I did not investigate it further but I noticed that when joining or reopening the app I can see that dialer receives only peerId instead of whole peer address:

```
connections-manager.service, payload peers: QmUUJEBNZaPb2xvfciWRKB421qdmTY7cg7xNVGNLEXcUUt
      at Socket.<anonymous> (src/utils.ts:149:15)
  console.log
    stdout:
    connections-manager.service before createLibp2pInstance [ 'QmUUJEBNZaPb2xvfciWRKB421qdmTY7cg7xNVGNLEXcUUt' ]
      at Socket.<anonymous> (src/utils.ts:149:15)
  console.log
    [e2e_534h3hm7rbk0]: 2023-11-02T12:07:46.777Z backend:Libp2pService Initializing libp2p for QmeXtiFNBQMyZ6KXnyiKHtF6HevBSQZu12Dbo2jfrcZ4nr, bootstrapping with 1 peers
      at Socket.<anonymous> (src/utils.ts:159:15)
  console.log
    [e2e_534h3hm7rbk0]: 2023-11-02T12:07:46.777Z backend:ProcessInChunks Processing 1 items
    2023-11-02T12:07:46.778Z backend:ProcessInChunks Processing QmUUJEBNZaPb2xvfciWRKB421qdmTY7cg7xNVGNLEXcUUt failed, message: multiaddr "QmUUJEBNZaPb2xvfciWRKB421qdmTY7cg7xNVGNLEXcUUt" must start with a "/"
```

It doesn't seem like a serious issue because libp2p eventually connects (maybe uses its internal addressBook, I don't know).

**Comment by holmesworcester** [comment]
How much of a delay do we think this causes?

**Comment by holmesworcester** [comment]
Note: we think this is already fixed in the branch with libp2p fixes.

**Comment by EmiM** [comment]
> How much of a delay do we think this causes?

I don't think it causes a delay because before dialing peer we convert peer address string to multiaddr and it immediately screams that the address is incorrect.
Although as I think about it more it could cause a delay - if we dial peers in chunks and part of the chunk is invalid then we really just dial a few peers at once (less then we could).

* Why we have only peers in the list of addresses? (error explanation)
During nest(?) refactoring saving peer stats format to leveldb changed -> before key in saved object was a full peer address, after refactoring it became a peerID. This led to the bugs in other parts of the code.

**Comment by holmesworcester** [comment]
Okay, I get it. Why is this in "blocked"?

**Comment by EmiM** [comment]
I don't know why you put it to blocked.
It will be fixed as a part of #1982

**Comment by kingalg** [comment]
Version: 2.0.3-alpha.10

Fixed as a part of #1982
